app/services/acceptable_verified_credential_adoption_service.rb

```diff
@@ -49,4 +50,15 @@ def verified_credential_at?
     user_avc = UserAcceptableVerifiedCredential.find_by(user_account: user.user_account)
     user_avc&.acceptable_verified_credential_at || user_avc&.idme_verified_credential_at
   end
+
+  def log_results(conversion_type)
+    StatsD.increment("#{stats_key}.conversion_attempt", tags: [
+                       "conversationType:#{conversion_type}",
+                       "credentialType:#{credential_type}"
+                     ])
+  end
+
+  def stats_key
+    'api.user_transition_availability'
+  end
```

Comment:
I would suggest putting the `conversion_type` and `credential_type` as namespaces in your key and not worrying about tags. 
Something like: `api.user_transition_availability.organic_modal.ds_logon` etc. 

Tags are used more for metadata fields such as `env`, `host`, etc (which are auto populated).

Reply:
Good to know! Thank you! I will make that change!

Reply:
@rileyanderson Would you mind giving it a second review when you get a chance? Thanks! 🙇‍♀️